app/subscriber/src/components/modal/Modal.tsx

```diff
@@ -90,7 +90,7 @@ export const Modal: React.FC<IModalProps> = ({
                       <Button
                         variant={type === 'delete' ? 'error' : 'primary'}
                         onClick={onConfirm}
-                        disabled={isSubmitting && !enableConfirm}
+                        disabled={isSubmitting || !enableConfirm}
```

Comment:
If either then disable.  The way this was implemented didn't make sense.  I hope where ever else it's used it doesn't break.